packages/ui/src/proposals/modals/AddNewProposal/AddNewProposalModal.tsx

```diff
@@ -55,7 +55,7 @@ export type BaseProposalParams = Exclude<
   string | Uint8Array
 >
 
-const isLastStepActive = (steps: Step[]) => {
+export const isLastStepActive = (steps: Step[]) => {
```

Comment:
So let's move it to `@/common/machine(?)/utils.ts` (or something similar). Wherever the `transactionMachine` is located.